server/pages/trackProgressPage.test.ts

```diff
@@ -1,106 +1,126 @@
-import TrackProgressPage from './trackProgressPage'
+import TrackProgressPage, { TrackProgressPageInput } from './trackProgressPage'
 
 describe('TrackProgressPage', () => {
-  it('returns validation errors', () => {
-    const page = new TrackProgressPage({})
+  describe('validationErrors', () => {
+    it('returns errors when date is empty', () => {
+      const page = new TrackProgressPage({} as TrackProgressPageInput)
 
-    expect(page.validationErrors()).toEqual({
-      team: { text: 'Choose a team' },
-      'startDate-day': { text: 'Include the start date' },
-      'endDate-day': { text: 'Include the end date' },
+      expect(page.validationErrors()).toEqual({
+        team: { text: 'Choose a team' },
+        'startDate-day': { text: 'From date must include a day, month and year' },
```

Comment:
did we say this could just be startDay and endDay now?

Reply:
When I used `startDate`, focus wasn't correctly assigned to the date input, but there may be another solution.